internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -90,11 +90,13 @@ If the argument holds an image tag, it must be present remotely.`,
 		"Disable resource cleanup after tests are run")
 	scorecardCmd.Flags().DurationVarP(&c.waitTime, "wait-time", "w", 30*time.Second,
 		"seconds to wait for tests to complete. Example: 35s")
+	// Use the digest of the latest scorecard-store image
 	scorecardCmd.Flags().StringVarP(&c.storageImage, "storage-image", "b",
-		"quay.io/operator-framework/scorecard-storage:latest",
+		"quay.io/operator-framework/scorecard-storage@sha256:f7bd62664a0b91034acb977a8bb4ebb76bc98a6e8bdb943eb84c8e364828f056",
```

Comment:
Hi @tonyskapunk thank you for the fix! 

Let's add a small comment explaining that operator-sdk + preflight + DCI integration in a disconnected environment requires these two images to be referred by digests and not by tags, to avoid the second occasional revert in the future.